physical_properties/pKa/example_submission_files/typeII-MehtapIsik-1.csv

```diff
@@ -0,0 +1,63 @@
+# Submission Type II
+#
+# PREDICTION SECTION
+#
+# Fill one typeII_microstate_fractional_populations.csv template file for all molecules and microstates you have predictions for.
+# For each molecule, report as many microstates as your method predicts.
+# Add new lines for each predicted microstate.
+# Microstate ID of the predicted microstate should be recorded in the first column.
+# Consecutive columns must be used to report ln(fractional microstate population) values of your predicted microstate at the pH value indicated by column title.
+# e.g. In column 3 report ln(fractional microstate population) at pH 2.10 of your predicted microstate.
+# Use scientific notation with 3 decimals of precision (e.g. -2.30e0 or 3.14e-4).
+# For microstates predicted to have 0 populations, instead of ln(0), report as "-infinity".
+# Do not report error estimates in this section. You can report error estimates in Method secton.

```

Comment:
If I read this I would provide the same lines as below with uncertainty estimates for each measurement. Is that what you're expecting people to do? If you want people to have the option of reporting uncertainties maybe you should make an `uncertainties:` section for this file type.

Reply:
We don't plan to evaluate uncertainties of this submission type.

Reply:
OK, maybe just say that and not tell people they can report error estimates in the method section?

Reply:
OK. This is reasonable.

Reply:
I removed " You can report error estimates in Method section." sentence from the template file.